spec/ruby/core/numeric/shared/step.rb

```diff
@@ -0,0 +1,409 @@
+require File.expand_path('../../../../spec_helper', __FILE__)
+require File.expand_path('../../fixtures/classes', __FILE__)
+
+describe :numeric_step, :shared => true do
+
+  def get_args(*args)
```

Comment:
The get_args method is new and it's specific to step method specs with kw arguments. It allows to run the tests with different argument passing strategies.
My idea would be to move it into a into a specific helper file. What do you think?